### ISSUE TITLE
Installing using Gomod returns a different version of Window interface

### ISSUE THREAD

**Issue by sitompul** [fyne-io/fyne#350]
**Environments**:

- Version: 1.0.1
- Debian Testing 10
- Golang 1.12.6

**Goal**: Running using Window.SetMainMenu() and fyne.NewMainMenu

**Problem**: SetMainMenu and NewMainMenu functions are undefined.

**What i've tried**: 

- Running regular window.SetContent() successfully
- Checking my vendor folder, it looks like that the window interface shows a different form from the one in github. The one inside my vendor folder lacks some interface methods, eg SetMainMenu.

**My build log**:
```
# command-line-arguments
cmd/app/main.go:15:3: w.SetMainMenu undefined (type fyne.Window has no field or method SetMainMenu)
cmd/app/main.go:15:16: undefined: fyne.NewMainMenu
cmd/app/main.go:15:33: undefined: fyne.NewMenu
cmd/app/main.go:16:3: undefined: fyne.NewMenuItem
```

**Source**:

```go
package main

import (
	"fmt"

	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
	"com/myapp/configs"
)

func main() {
	a := app.New()
	w := a.NewWindow(configs.AppName)
	w.SetMainMenu(fyne.NewMainMenu(fyne.NewMenu("File",
		fyne.NewMenuItem("New", func() {
			fmt.Println("Test")
		}),
	)))

	w.SetContent(widget.NewVBox(
		widget.NewLabel("Hello This is App!"),
	))

	w.ShowAndRun()
}

```

**Comment by andydotxyz** [comment]
You’ll need to use version 1.1.0 for the menu functionality. Apologies that wasn’t clear from the documentation.

**Comment by sitompul** [comment]
@andydotxyz Will the version 1.1.0 be released? or should i install from v 1.1.0 branch manually?

**Comment by andydotxyz** [comment]
Apologies, we wrapped up 1.1 last week but it seems the tag did not push correctly.
This should now work as expected.
